docs/virtual_machines/templates.md

```diff
@@ -93,8 +93,8 @@ the following minimal snippet highlights the fields which you can edit:
 
 ### Relationship between templates and VMs
 
-Once
-[processed](https://docs.openshift.com/enterprise/3.0/dev_guide/templates.html#creating-from-templates-using-the-cli),
+Once 
+[processed](https://docs.okd.io/latest/openshift_images/using-templates.html), 
```

Comment:
This link doesn't pertain to the content.  I think we should just fix the OpenShift link.